twikiget/cli.py

```diff
@@ -55,3 +72,102 @@ def wget_warc(url, wget_options, out_warc_file, directory_prefix):
                       directory_prefix=directory_prefix,
                       include=include)
               )
+
+
+@twikiget.command('list')
+@click.option(
+    '--json',
+    'output_format',
+    flag_value='json',
+    default=None,
+    help='Get output in JSON format.')
+@click.option(
+    '--content-type',
+    'filter_content_type',
+    default='',
+    help='Filter files in an archive by content_type. '
+         'It can be either full version or just a begging of type name')
+@click.argument('warc_file')
+def list(warc_file, output_format, filter_content_type):
+    """ List files in WARC archive.
+
+    The list can be filtered by the HTTP Content Type,
+     and exported as json if needed.\n
+    Note that `content-type` option can be a full name of a type
+     or, to search broader, just the first part of it
+     e.g. `text/css` and `text`
+
+    Examples::
```

Comment:
Note that the display is broken regarding new lines, you may need to use `\b` or `r` as we do in REANA.

![2019-07-15-184152_689x314_scrot](https://user-images.githubusercontent.com/517546/61233212-4fc39f80-a730-11e9-85f6-f70272466d42.png)